Pod/Core/UnusedStepsTracker.m

```diff
@@ -56,9 +56,9 @@ - (void)testBundleDidFinish:(NSBundle *)testBundle {
     if (self.bundleCounter == 0) {
         NSMutableSet *unusedSteps = self.allSteps;
         [unusedSteps minusSet: self.executedSteps];
-        if (unusedSteps.count > 0) {
-            self.printUnusedSteps(unusedSteps.allObjects);
-        }
+        // if (unusedSteps.count > 0) {
```

Comment:
```suggestion
        // We don't need this since it just checks the steps used in the test run,
        // so if only running 1 test, a lot of steps will appear here.
        // if (unusedSteps.count > 0) {
```